docs/resources/Guild.md

```diff
@@ -460,3 +475,127 @@ Returns the [guild embed](#DOCS_GUILD/guild-embed-object) object. Requires the '
+| changes | array of [audit log change object](#DOCS_GUILD/audit-log-change-object) | changes made to the target id|
+| user_id | snowflake | the user who made the changes
+| id | snowflake | id of the entry
+| action_type | an [audit log event](#DOCS_GUILD/audit-log-event) | type of action that occured
+| options | object | additional information specific to the action type
+
+###### Audit Log Events
+
+| Event | Value |
+|-------|------|
+| GUILD_UPDATE | 1 |
+| CHANNEL_CREATE | 10 |
+| CHANNEL_UPDATE | 11 |
+| CHANNEL_DELETE | 12 |
+| CHANNEL_OVERWRITE_CREATE | 13 |
+| CHANNEL_OVERWRITE_UPDATE | 14 |
+| CHANNEL_OVERWRITE_DELETE | 15 |
+| MEMBER_KICK | 20 |
+| MEMBER_PRUNE | 21 |
+| MEMBER_BAN_ADD | 22 |
+| MEMBER_BAN_REMOVE | 23 |
+| MEMBER_UPDATE | 24 |
+| MEMBER_ROLE_UPDATE | 25 |
+| ROLE_CREATE | 30 |
+| ROLE_UPDATE | 31 |
+| ROLE_DELETE | 32 |
+| INVITE_CREATE | 40 |
+| INVITE_UPDATE | 41 |
+| INVITE_DELETE | 42 |
+| WEBHOOK_CREATE | 50 |
+| WEBHOOK_UPDATE | 51 |
+| WEBHOOK_DELETE | 52 |
+| EMOJI_CREATE | 60 |
+| EMOJI_UPDATE | 61 |
+| EMOJI_DELETE | 62 |
+| MESSAGE_DELETE | 72 |
+
+### Audit Log Change Object
+
+###### Audit Log Change Structure
+
+| Field | Type | Description |
+|-------|------|-------------|
+| new_value | string/int/bool | new value of the key |
+| old_value | string/int/bool | old value of the key |
+| key | string | type of audit log [change key](#DOCS_GUILD/audit-log-change-key)
+
+###### Audit Log Change Key
+| Name | Type | Description |
+| ---- | --- | ----------- |
+| name | string | guild name changed
+| icon_hash | string | guild icon changed
+| splash_hash | string | guild invite splash page artwork changed
+| owner_id | snowflake | guild owner changed
+| region | string | voice region changed
+| afk_channel_id | snowflake | afk channel changed
+| afk_timeout | int | afk timout duration changed
+| mfa_level | int | two-factor auth requirement changed
+| widget_enabled | bool | widget (bot, webhook) added or removed from a channel
+| widget_channel_id | snowflake | channel id to which a widget was added or from which one was removed
+| verification_level | int | required verification level changed
+| explicit_content_filter | int | change in [whose messages](#DOCS_GUILD/explicit-content-filter-level) are scanned and deleted for explicit content in the server
+| default_message_notifications | int | default [message notification level](#DOCS_GUILD/default-message-notification-level) changed|
+| vanity_url_code | string | guild invite vanity url changed
+| position | int | text or voice channel position changed
+| topic | string | text channel topic changed
+| type | string/int | type of entity created - can be of [channel type](#DOCS_CHANNEL/channel-types) or a string like "role"|
+| bitrate | int | voice channel bitrate changed
+| permission_overrides | array of [channel overwrite](#DOCS_CHANNEL/overwrite-object) objects| permissions on a channel changed |
```

Comment:
I'll swap it to `overwrites` as that's what comes in the response object, and will investigate the discrepency